infrastructure/data_access/http/synthetics_repo.py

```diff
@@ -85,6 +94,7 @@ def transform_to_internal_mesh_columns(input_columns: List[V202101beta1MeshColum
                 value=scale_to_percents(input_column.metrics.packet_loss.value),
             ),
             health=transform_to_internal_health_items(input_column.health),
+            last_updated_utc=datetime.now(timezone.utc),
```

Comment:
The timestamp must be extracted from the health data, so that it reflects the time when the observation was collected (as opposed the time when the data was fetched from Kentik). The observation time is much more relevant for customers.

Reply:
Renaming last_updated_utc to utc_timestamp.
`utc_timestamp=input_column.metrics.time`
NOTE: input_column.metrics.time seems to carry the beginning of the requested time window, not the end, not sure why is that?

Reply:
Hmm, yes the `mesh` seems to contains only the oldest sample in the requested time period. I'd expect it to contain the newest available sample (or all available samples) :-/. I'll bring up the counterintuitive behavior with engineering. 
However, samples from the entire requested interval are in the `health[0].tasks[].agents[].health` list in the response data. Considering that we will soon need to split fetching recent matrix data from time-series history for individual connections, we can request only time period going back about 2 test periods when fetching matrix.